### DISCUSSION TITLE
Justify text

### DISCUSSION THREAD

**Discussion by wesleigomes** [QuestPDF/QuestPDF#89] [Q&A] [answered]
Hi, 

Is there a way to justify text? I really this function.

**Comment by MarcinZiabek** [comment, marked as answer]
Hello 😀

Justify text is currently not supported. It is possible to align to the left, to the right or centre text. 

This feature is planned to be available in one of the future releases.

**Reply by wesleigomes** [reply]
Thank you!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Text justification is now fully supported via the `.Justify()` method. You can find the documentation and examples here:
https://www.questpdf.com/api-reference/text/paragraph-style.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
